.baseline/findbugs/excludeFilter.xml

```diff
@@ -1,5 +1,9 @@
 <!-- See http://findbugs.sourceforge.net/manual/filter.html for syntax -->
 <FindBugsFilter>
+    <!-- ignore generated code -->
+    <Match>
+        <Class name="~.*\.*Immutable.*" />
```

Comment:
Baseline is on 0.2.2 -- bumping to 0.2.4 locally didn't help.

We're using version 1.1 of `gradle-processors`, and looking at the version history I don't think there are any changes between 1.1 and 1.1.5 (current release) that would fix this issue.